backend/user_manager.js

```diff
@@ -0,0 +1,185 @@
+
+const mysql = require('mysql');
+const session = require('express-session');
+const MySQLStore = require('express-mysql-session')(session);
+const md5 = require('md5');
+import mysqlUtils from './mysql_utils';
```

Comment:
Please verify if we can use the `require` syntax here too.

Reply:
yes they work too.